Index: xamarin-pkg-nuget/src/CommandLine/CommandLine.csproj
===================================================================
--- xamarin-pkg-nuget.orig/src/CommandLine/CommandLine.csproj	2015-09-02 16:08:43.144198018 +0100
+++ xamarin-pkg-nuget/src/CommandLine/CommandLine.csproj	2015-09-02 16:08:43.140198017 +0100
@@ -32,7 +32,7 @@
     <Reference Include="Microsoft.CSharp" />
     <Reference Include="Microsoft.Web.XmlTransform, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\lib\Microsoft.Web.XmlTransform.dll</HintPath>
+      <HintPath>..\..\xdt\XmlTransform\bin\Debug\Microsoft.Web.XmlTransform.dll</HintPath>
     </Reference>
     <Reference Include="System" />
     <Reference Include="System.ComponentModel.Composition" />
Index: xamarin-pkg-nuget/src/Core/Core.csproj
===================================================================
--- xamarin-pkg-nuget.orig/src/Core/Core.csproj	2015-09-02 16:08:43.144198018 +0100
+++ xamarin-pkg-nuget/src/Core/Core.csproj	2015-09-02 16:08:43.140198017 +0100
@@ -222,7 +222,7 @@
     <Compile Include="Utility\WebRequestEventArgs.cs" />
     <Compile Include="FileModifiers\XdtTransformer.cs" />
     <Reference Include="Microsoft.Web.XmlTransform">
-      <HintPath>..\..\lib\Microsoft.Web.XmlTransform.dll</HintPath>
+      <HintPath>..\..\xdt\XmlTransform\bin\Debug\Microsoft.Web.XmlTransform.dll</HintPath>
     </Reference>
     <Reference Include="System" />
     <Reference Include="System.ComponentModel.DataAnnotations" />
